frontends/ol-components/src/components/ChoiceBox/ChoiceBoxField.tsx

```diff
@@ -0,0 +1,113 @@
+import React from "react"
+import FormControl from "@mui/material/FormControl"
+import FormGroup from "@mui/material/FormGroup"
+import FormLabel from "@mui/material/FormLabel"
+import Grid, { type GridProps } from "@mui/material/Grid"
+import { ChoiceBox } from "./ChoiceBox"
+import type {
+  ChoiceBoxGridProps,
+  ChoiceBoxChoice,
+  ChoiceBoxProps,
+} from "./ChoiceBox"
+
+interface BaseChoiceBoxFieldProps extends ChoiceBoxGridProps {
+  label: React.ReactNode
+  choices: ChoiceBoxChoice[]
+  onChange: ChoiceBoxProps["onChange"]
+  className?: string
+}
+
+interface ChoiceBoxFieldProps extends BaseChoiceBoxFieldProps {
+  type: ChoiceBoxProps["type"]
+  name?: string
+  isChecked: (choice: ChoiceBoxChoice) => boolean
```

Comment:
Comment: I can imagine `isDisabled` would also be useful, but I don't think we need to add it now.